src/redis.ts

```diff
@@ -26,4 +23,7 @@ connectionObj = {
 }
 
 export const redis = new Redis(connectionObj);
+redis.on('error', (err) => baseLogger.fatal("Redis error", err))
```

Comment:
the way the pino logger works is that the message should be the last argument.
ie: https://github.com/pinojs/pino/blob/master/docs/api.md#error

I don't think we need to have a fatal for this. error is probably good.